src/components/hero.jsx

```
@@ -75,61 +75,61 @@
+          { searchTerm.length > 1 && <div className={showDropdown ? 'flex flex-wrap justify-center gap-4 mt-[-20px] mb-5 p-2.5 bg-white border border-gray-300 rounded-2xl max-w-[800px]' : 'hidden'}>
+            {data.length > 0 && data
+              .slice(0, 20)
+              .map((item) => (
+                <div
+                  onClick={() => {
+                    setSearchTerm(item.name);
+                    setSelectedBreedId(item.id);
+                  }}
+                  className={data.length > 1 ? "bg-white text-yellow-500 border border-yellow-500 rounded-2xl p-2.5 cursor-pointer transition-colors duration-300 hover:bg-yellow-300 hover:text-white" : "bg-yellow-500 text-white border border-yellow-500 rounded-2xl p-2.5 cursor-pointer transition-colors duration-300"}
+                  key={item.name}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)